functions/influxdb+grafana.bash

```diff
@@ -286,11 +286,18 @@ influxdb_install() {
 ##
 
 grafana_install(){
+  local grafana_first_time
   echo -n "Installing Grafana... "
   cond_redirect wget -O - https://packages.grafana.com/gpg.key | apt-key add - || FAILED=2
   echo "deb https://packages.grafana.com/oss/deb stable main" > /etc/apt/sources.list.d/grafana.list || FAILED=2
   cond_redirect apt-get update || FAILED=1
   cond_redirect apt-get -y install grafana || FAILED=2
+  if [ -e /var/lib/grafana/grafana.db ]; then
+    echo -n "Detected existing Grafana install..."
+  else
+    # shellcheck disable=SC2034
+    grafana_fist_time=true
```

Comment:
"fist"
-> does the code work with that or is the check inverted, too ??
shellcheck hint probably not needed as well

Reply:
too late for coding :-/ shellcheck is always right, I should have known
done.

Reply:
> too late for coding :-/

Move on to some reviewing then ;-)
#861, #830, #807, and most important #882 
Juggling with too many open PRs can easily end up in chaos